Add info about gwlevels service (#160)
DOI-USGS/dataretrieval-python: README.md

# dataretrieval: Download hydrologic data

:warning: USGS data availability and format are changing on Water Quality Portal (WQP). Since March 2024, data obtained from WQP legacy profiles will not include new USGS data or recent updates to existing data. 
To view the status of changes in data availability and code functionality, visit: https://doi-usgs.github.io/dataRetrieval/articles/Status.html

> :mega: **09/03/2024:** The groundwater levels service has switched endpoints due to WaterServices database updates. `dataretrieval` was updated accordingly in [`v1.0.10`](https://github.com/DOI-USGS/dataretrieval-python/releases/tag/v1.0.10), but older versions of the package will return 503 errors from the discontinued service when using `nwis.get_gwlevels` or the `service='gwlevels'` argument. Visit this Water Data For the Nation [blog post](https://waterdata.usgs.gov/blog/wdfn-waterservices-2024/) for more information.

Reviewer: ```suggestion
:mega: **09/03/2024:** The groundwater levels service has switched endpoints, and `dataretrieval` was updated accordingly in [`v1.0.10`](https://github.com/DOI-USGS/dataretrieval-python/releases/tag/v1.0.10). Older versions using the discontinued endpoint will return 503 errors for `nwis.get_gwlevels` or the `service='gwlevels'` argument. Visit [Water Data For the Nation](https://waterdata.usgs.gov/blog/wdfn-waterservices-2024/) for more information.
```
Your call here. Mine is a bit tighter, IMO.
